sdk/src/test/java/com/google/cloud/dataflow/sdk/io/PubsubIOTest.java

```diff
@@ -70,29 +73,13 @@ public void testTopicValidationSuccess() throws Exception {
   }
 
   @Test
-  public void testTopicValidationBadCharacter() throws Exception {
```

Comment:
Not sure I like this change. Given an SVP, we should be able to catch these bad names, shouldn't we?